### ISSUE TITLE
It doesn't work in blender 2.90.1. Please update it for new blender versions.

### ISSUE THREAD

**Issue by whyAREyouRUNNING-debug** [taniwha/io_object_mu#45]

**Comment by Massetstephane** [comment]
hi !
tested with blender 2.90.1
the result here [https://forum.kerbalspaceprogram.com/index.php?/topic/198068-tank-kerbal-space-program-modding-tutorial-using-blender-only-by-kerbonerd-with-example-file-for-blender/](url)
it's work well !

**Comment by StoneBlue** [comment]
Yes... I've been using it all the way from 2.79 to 2.92... have also been using it in BforArtists since their 2.80 fork, all the way up to the present 2.92 fork
Taniwha has been constantly updating and adding new features for quite awhile.

If there are specific functions not working for you, adding more detail to your Issue would probably help a lot.

**Comment by Camelhairs06** [comment]
@StoneBlue 
I've copied The folder into my Blender addons folder however I cannot find it in my addon section in preferences.
I'm running version 2.91.

**Comment by StoneBlue** [comment]
I have no real idea if Steam could be the issue. I recently got Steam myself, and bought a game which I tried installing mods for, and I found Steam to be a horrible complicated "middle-man".

I wouldnt suggest peoplpe use Steam for anything more than they absolutely *have* to use it for.

All I can tell you is, and it may not be the correct way, but its seemed to have worked for me, with installing Blender using the executable installer, is to copy the addon folder to the /scripts/addons_contrib folder in your /Program Files/Blender folder.
